features/figure.yml

```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
 name: <figure>
-description: The `<figure>` element TODO.
+description: The `<figure>` HTML element represents self-contained content, potentially with an optional `<figcaption>`.
```

Comment:
I think this would benefit from being more concrete, like "picture with text under it" concrete, even `<figure>` can be used for other things too. I'll leave it to @ddbeck to suggest better wording though :)

Reply:
How about this?

```suggestion
description: The `<figure>` and `<figcaption>` HTML elements represent an illustration, diagram, text, or other self-contained reference content, with an optional caption.
```